powersimdata/data_access/data_access.py

```diff
@@ -180,21 +219,41 @@ def move_to(self, file_name, to_dir, change_name_to=None):
         :param str change_name_to: new name for file when copied to data store.
         """
         self._check_filename(file_name)
-        src = posixpath.join(server_setup.LOCAL_DIR, file_name)
+        src = os.path.join(server_setup.LOCAL_DIR, file_name)
         file_name = file_name if change_name_to is None else change_name_to
-        dest = posixpath.join(self.root, to_dir, file_name)
+        dest = os.path.join(self.root, to_dir, file_name)
         print(f"--> Moving file {src} to {dest}")
         self._check_file_exists(dest, should_exist=False)
-        self.copy(src, dest)
```

Comment:
Avoiding the use of `copy` because `dest` is a file, and we don't it to automatically create `/path/to/file.txt/` as a folder